crates/schema-impl/src/json_key_conversion.rs

```diff
@@ -325,36 +326,36 @@ mod tests {
     }
 
     json_tests!(string);
-    json_tests!(bytes);
-    json_tests!(number);
-    json_tests!(nested_message, nested_key_structure());
-    json_tests!(
-        test: nested_message_with_default,
-        field_name: nested_message,
-        key_structure: nested_key_structure(),
-        test_message: TestMessage {
-            nested_message: Some(NestedKey {
-                b: "b".to_string(),
-                ..Default::default()
-            }),
-            ..Default::default()
-        }
-    );
-    json_tests!(
-        test: double_nested_message,
-        field_name: nested_message,
-        key_structure: nested_key_structure(),
-        test_message: TestMessage {
-            nested_message: Some(NestedKey {
-                b: "b".to_string(),
-                other: Some(OtherMessage {
-                    d: "d".to_string()
-                }),
-                ..Default::default()
-            }),
-            ..Default::default()
-        }
-    );
+    // json_tests!(bytes);
+    // json_tests!(number);
+    // json_tests!(nested_message, nested_key_structure());
+    // json_tests!(
+    //     test: nested_message_with_default,
+    //     field_name: nested_message,
+    //     key_structure: nested_key_structure(),
+    //     test_message: TestMessage {
+    //         nested_message: Some(NestedKey {
+    //             b: "b".to_string(),
+    //             ..Default::default()
+    //         }),
+    //         ..Default::default()
+    //     }
+    // );
+    // json_tests!(
+    //     test: double_nested_message,
+    //     field_name: nested_message,
+    //     key_structure: nested_key_structure(),
+    //     test_message: TestMessage {
+    //         nested_message: Some(NestedKey {
+    //             b: "b".to_string(),
+    //             other: Some(OtherMessage {
+    //                 d: "d".to_string()
+    //             }),
+    //             ..Default::default()
+    //         }),
+    //         ..Default::default()
+    //     }
+    // );
```

Comment:
Given that we are linking this PR, should we remove the code instead of commenting it?